src/webgpu/compat/api/validation/encoding/programmable/pipeline_bind_group_compat.spec.ts

```diff
@@ -329,6 +330,13 @@ Tests that you can not use 2 different views of the same texture in a render pas
   )
   .fn(t => {
     const { encoderType, bindCase, useCase, textureType } = t.params;
+
+    t.skipIf(
+      t.isCompatibility &&
+        textureType === 'storage' &&
+        !(t.device.limits.maxStorageBuffersInFragmentStage! > 2)
+    );
```

Comment:
nit: better have a message like the other pull requests indicating reason for skipping